common/src/main/java/com/genexus/diagnostics/UserLog.java

```diff
@@ -120,4 +120,51 @@ public static void debug(String message, String topic) {
 	public static void debug(String message, String topic, Throwable ex) {
 		getLogger(topic).debug(message, ex);
 	}
+
+	public static void setContext(String key, Object value) {
+		// Topic is ignored, also if you put something
+		getLogger("$").setContext(key, value);
+	}
+
+	public static void write(String message, String topic, int logLevel, Object data, boolean stackTrace) {
+		getLogger(topic).write(message, logLevel, data, stackTrace);
+	}
+
+	public static void write(String message, String topic, int logLevel, Object data) {
+		getLogger(topic).write(message, logLevel, data, false);
```

Comment:
I think it's better to call write(message, logLevel, data, false); here to avoid duplicating code.